### PR TITLE
Create minimum repository documentation

### PR DESCRIPTION
**Pull request by JackEAllen**
ISSUE: [#6903](https://github.com/ISISComputingGroup/IBEX/issues/6903)

Add minimal documentation to repository README.md containing useful recourse links and commands to speed up development. 

Having some documentation inside the README is a choice agreed upon during retrospective meeting which notes can be found [HERE](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Retrospective-notes-2021.11.17)

To review:
- [ ] README.md is well structured and contains or links to all useful information that a developer may need to quickly get started working on an issue in the repository, minimising the need to leave the IDE.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+To start EPICS environment locally, simply run the `config_env` script: 
+`C:\Instrument\Apps\EPICS\config_env`
+
+When the IOC is running, you can use the following command to checks record values:
+`<epics command> %mypvprefix%PEARLPC_01:<PV record>`
+
+To test the IOC using LEWIS emulator, execute the `run_test.bat` script located:
+`master\system_tests\run_tests.bat`
+
+Add `-a` flag to `run_test.bat` to run IOC emulator and not tests straight away if wishing to view in IBEX or check PV values when testing.
```
Comment (JamesKingWork) [inline README.md:35]: We should link to the ioctestframework readme as this will get out of date easily

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+`C:\LabVIEW Modules\Instruments\PEARL\PEARL Pressure Cell Controller`
+
+### WIKI Pages: 
+
+https://github.com/ISISComputingGroup/IBEX/wiki/PEARL-Instrument-Details
+
+### IOC Repository: 
+
+https://github.com/ISISComputingGroup/EPICS-ioc/tree/4cd6b6a481666ea77c155126b5114b4ca3714eb3/PEARLPC
```
Comment (JamesKingWork) [inline README.md]: Why this commit and not master?

**Review by JamesKingWork — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JackEAllen) [reply README.md:35]: Isn't this repo following the new format where tests are located next to emulator and DB files? I was under the impression this was the new format that we planned to stay with and as such shouldn't get out of date?

File: README.md (hunk 2 above)
Comment (JackEAllen) [reply README.md]: Updated to master branch. This is something I missed as I just copied the URL I was take too when navigating from https://github.com/ISISComputingGroup/EPICS/tree/master/ioc into IOC sub-module.

**Pull request merged**